oracle-linux-image-tools/distr/ol7-slim/ol7-ks.cfg

```diff
@@ -73,7 +74,6 @@ kpartx
 net-tools
 iptables-services
 btrfs-progs
-bind-utils
```

Comment:
Why are you removing `bind-utils`, out of curiousity?

Reply:
It isn't currently installed on any box an nobody complained so far ;)

Reply:
Yeah, but you specifically make a choice here to remove it. What was the rationale behind removing the line?

Reply:
For now, building the the exact same boxes we are releasing.